docs/generative_ai.md

```diff
@@ -31,7 +31,7 @@ Human: Answer all the questions below as a json object with key value pairs, the
 <br><transcript> 
 <br>{transcript} 
 <br></transcript> 
-<br>Assistant: Here is the JSON object with the answers to the questions:
+<br>Assistant: 
```

Comment:
Per Anthropic docs, we should have double newlines.. `\n\nAssistant:` to align with the model's training.  Or in our templates, double <br> tags. 
See other slack messages.. with the dbl newline it doesn't complain about additional text..